ztp/siteconfig-generator/siteConfig/siteConfigHelper.go

```diff
@@ -0,0 +1,129 @@
+package siteConfig
+
+import (
+	"encoding/json"
+	"fmt"
+	"log"
+
+	machineconfigv1 "github.com/openshift/machine-config-operator/pkg/apis/machineconfiguration.openshift.io/v1"
+	mcfgctrlcommon "github.com/openshift/machine-config-operator/pkg/controller/common"
+
+	"github.com/lack/yamltrim"
+	yaml "gopkg.in/yaml.v3"
+)
+
+const mcRoleKey = "machineconfiguration.openshift.io/role"
```

Comment:
I recently learned we can use `machineconfigv1.MachineConfigRoleLabelKey` instead of copying this value around by hand all the time :)

Reply:
Thank you for your insight.